website/docs/advanced-concepts/granular-access-control/README.md

```diff
@@ -48,6 +48,18 @@ A group is a collection of users that allows you to assign roles to multiple use
 When you share an application or workspace, you can invite a group and assign the desired role to it. This makes it easy to grant permissions to multiple users at once, and you can modify the group's roles to update the permissions for all its members simultaneously.
 
 
+## Programmatic access control
+
+You can utilize roles and groups assigned to a user to programmatically manage access to various entities such as widgets, datasources, APIs, and queries. 
```

Comment:
Suggestion for rewriting this section as below

:::info 
This feature is only available on the [Business Edition](https://www.appsmith.com/pricing).
:::

You can programmatically manage access to various entities such as widgets, datasources, and queries by accessing the roles and groups assigned to the logged-in user. 

To turn on this feature, 

1. On the homepage, go to **Admin Settings > General**.
2. Under the **Programmatic Access Control** section, select the **Access roles and user groups in code for conditional business logic** checkbox.  [Screenshot of the setting page below this point]
3. To manage access control programmatically, use the `roles` and `groups` properties of the Appsmith user object `appsmith.user` in JavaScript code to obtain an array of roles or groups assigned to the logged-in user. For more information on the object properties, refer to [roles](/reference/appsmith-framework/context-object#roles) and [groups](/reference/appsmith-framework/context-object#groups).

Reply:
The business edition info is already mentioned at the beginning of the page. Programmatic access control is an extension of GAC. I don't think we. need to mention it again.

Reply:
I think "refer to roles and groups" can get a little confusing. We should add something like - For more information on the object properties, refer to Appsmith user object.